RELEASE_TEMPLATE.md

```diff
@@ -48,19 +48,19 @@ pip install typedb-driver=={version}
 ### NodeJS driver
 
 NPM package: https://www.npmjs.com/package/typedb-driver
-Documentation: https://typedb.com/docs/clients/nodejs-driver
+Documentation: https://typedb.com/docs/drivers/nodejs/overview
 
 ```
-npm install typedb-driver@
+npm install typedb-driver@{version}
 ```
 
 ### C++ driver
 
-Compiled distributions comprising headers and shared libraries available at: https://github.com/vaticle/typedb-driver/releases/tag/{version}
+Compiled distributions comprising headers and shared libraries available at: https://cloudsmith.io/~typedb/repos/public-release/packages/?q=name%3A%5Etypedb-driver-cpp+version%3A{version}
 
 ### C driver
 
-Compiled distributions comprising headers and shared libraries available at: https://github.com/vaticle/typedb-driver/releases/tag/{version}
-
+Compiled distributions comprising headers and shared libraries available at: https://cloudsmith.io/~typedb/repos/public-release/packages/?q=name%3A%5Etypedb-driver-clib+version%3A{version}
+Documentation: https://typedb.com/docs/drivers/cpp/overview
```

Comment:
The download links will lead to the search page which will list all platform-specific libraries of specified language (C/C++) for the version given. Ex.: https://cloudsmith.io/~typedb/repos/public-release/packages/?q=name%3A^typedb-driver-clib+version%3A2.26.0

Reply:
Another option would be to specify all five platform-specific links that would lead to direct download, e.g. https://repo.typedb.com/public/public-release/raw/names/typedb-driver-clib-windows-x86_64/versions/2.26.0/typedb-driver-clib-windows-x86_64-2.26.0.zip